Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -113,13 +129,14 @@ def create_session(self, hops=0):
         # the settings dictionary
         settings['outgoing_port'] = 0
         settings['num_outgoing_ports'] = 1
+        extensions = list(DEFAULT_LT_EXTENSIONS)
 
         if hops == 0:
             settings['user_agent'] = 'Tribler/' + version_id
             # Elric: Strip out the -rcX, -beta, -whatever tail on the version string.
             fingerprint = ['TL'] + map(int, version_id.split('-')[0].split('.')) + [0]
             # Workaround for libtorrent 0.16.3 segfault (see https://code.google.com/p/libtorrent/issues/detail?id=369)
-            ltsession = lt.session(lt.fingerprint(*fingerprint), flags=1)
+            ltsession = lt.session(lt.fingerprint(*fingerprint), flags=0)
```

Comment:
Could you explain this change? Also, when you're at it, the oldest version of libtorrent we support is 0.16.18 so I'm not sure whether the workaround is still required. @qstokkink can you confirm this?

Reply:
@devos50 Do you want me to confirm the workaround or the fact we need a libtorrent version >= 0.16.18?

Reply:
The flags= parameter controlls what extensions are loaded into the lib torrent session. Since I've changed it to explicitly load the extensions, this changes the initially created session to load no extensions.

Reply:
@qstokkink whether the lib torrent version is at least 0.16.18 (I'm almost sure it is).

@Captain-Coder great!

Reply:
@devos50 it seems like Ubuntu 14 still ships `16.13`, but as I recall we dropped support for Ubuntu 14 anyway: so I'd say it's good.

Reply:
Let's remove this in another PR to devel then.